lib/view/list_widget.dart

```diff
@@ -139,6 +152,54 @@ class _ListWidgetState extends State<ListWidget> {
   List<Item> _ordered(Iterable<Item> items) =>
       items.sorted((a, b) => a.order - b.order);
 
+  Future<void> _exportAsMarkdown() async {
+    // We have to have separate exportAsMarkdown functions for mobile and desktop
+    // because the file_picker package (https://pub.dev/packages/file_picker) only
+    // supports saving files in desktop. This package seemed far superior to its competitors.
+    if (Platform.isAndroid || Platform.isIOS) {
+      await _exportAsMarkdownOnMobile();
+    } else {
+      await _exportAsMarkdownOnDesktop();
+    }
```

Comment:
The list widget really shouldn't need to worry about the platform differences. I would think all this logic should be contained inside the `ExportListAsMarkdownDialog` code

Reply:
Done